t/unit/tasks/test_result.py

```diff
@@ -389,12 +390,17 @@ def test_ready(self):
 
         assert not self.app.AsyncResult(uuid()).ready()
 
+    @pytest.mark.skipif(
+        platform.python_implementation() == "PyPy",
+        reason="Mocking here doesn't play well with PyPy",
```

Comment:
TBH I'm not sure why it fails only with PyPy, but I skip it to allow us to move forward.
Not sure if it's acceptable @auvipy

Reply:
I am goin to check it in  my work hours today.